lib/cirneco/doi.rb

```diff
@@ -1,5 +1,6 @@
 require 'thor'
 require 'bolognese'
+require 'csv'
```

Comment:
Is the `csv` library used?

Reply:
I forgot to remove this. I was thinking that maybe if the target was inside the file parsing csv would be useful, but the files we get do not have that.